android/src/main/java/io/sentry/react/RNSentryModuleImpl.java

```diff
@@ -731,10 +731,17 @@ public WritableMap stopProfiling() {
             return proguardUuid;
         }
         isProguardDebugMetaLoaded = true;
-        final @Nullable Properties debugMeta = (new AssetsDebugMetaLoader(this.getReactApplicationContext(), logger)).loadDebugMeta();
-        if (debugMeta != null) {
+        final @Nullable List<Properties> debugMetaList = (new AssetsDebugMetaLoader(this.getReactApplicationContext(),
+            logger)).loadDebugMeta();
+        if (debugMetaList == null) {
+            return null;
+        }
+
+        for (Properties debugMeta : debugMetaList) {
             proguardUuid = DebugMetaPropertiesApplier.getProguardUuid(debugMeta);
-            return proguardUuid;
+            if (proguardUuid != null) {
```

Comment:
if meta is available, do u expect this to be here? If so, worth sdk-debug logging?

Reply:
Yes, I do, I'll add a debug log.